schemas/oic.r.sacl.json

```diff
@@ -6,32 +6,23 @@
     "oic.r.sacl": {
       "type": "object",
       "properties": {
-        "aclist":  {
+        "aclist2":  {
           "type": "object",
           "description": "Access Control Entries in the Acl resource",
-          "$ref": "oic.sec.ace.json#/definitions/oic.sec.ace"
+          "$ref": "oic.sec.ace2.json#/definitions/oic.sec.ace2"
         },
         "signature":   {
           "type": "object",
           "description": "The signature over the ACL resource",
           "$ref": "oic.sec.sigtype.json#/definitions/oic.sec.sigtype"
-        },
-        "amsuuid": {
-          "$ref": "../../core/schemas/oic.types-schema.json#/definitions/uuid"
-        },
-        "ams":   {
-          "type": "object",
-          "description": "Entries in oic.sec.svc resource that manage access for the specified resource",
-          "$ref": "oic.sec.svctype.json#/definitions/oic.sec.svctype"
         }
       }
```

Comment:
Should this have an "rowneruuid" property since it no longer has an "amsuuid" or "ams"? Or is ownership implied somehow?